doc/man7/flux-environment.rst

```diff
@@ -344,7 +344,7 @@ components or writing tests.
    commands to resolve URIs to local form.  This is useful in test environments
    where the remote connector does not work.
 
-.. envvar:: FLUX_RESOURCE_STATUS_RPC
+.. envvar:: FLUX_RESOURCE_LIST_RPC
```

Comment:
Oops!

Reply:
I only noticed because I got it wrong in a couple places!